static/settings.ts

```diff
@@ -300,7 +299,7 @@ export class Settings {
             return {label: themes[theme].id, desc: themes[theme].name};
         });
         let defaultThemeId = themes.default.id;
-        if (window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches) {
+        if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia#browser_compatibility says that we _should_ be ok expecting this to always exist.

We can always enable it and monitor Sentry to see if this triggeres errors for any users